spec/models/local_authority_spec.rb

```diff
@@ -76,5 +76,13 @@
         expect(@local_authority.link_last_checked).to be_nil
       end
     end
+
+    describe "calculate_count_of_broken_links" do
```

Comment:
Were also thinking that we wouldn't need to define a local authority as you could use the `local_authority` association from the link factory, but maybe it's more readable? _shrug_